execution_engine_testing/tests/src/test/system_contracts/auction/bids.rs

```diff
@@ -3453,17 +3430,14 @@ fn should_handle_redelegation_to_inactive_validator() {
         .expect_success()
         .commit();
 
-    let step_request = StepRequestBuilder::new()
-        .with_parent_state_hash(builder.get_post_state_hash())
-        .with_protocol_version(ProtocolVersion::V1_0_0)
-        .with_next_era_id(builder.get_era().successor())
-        .with_reward_item(RewardItem::new(NON_FOUNDER_VALIDATOR_1_PK.clone(), 1))
-        .with_reward_item(RewardItem::new(NON_FOUNDER_VALIDATOR_2_PK.clone(), 1))
-        .build();
-
-    builder
-        .step(step_request)
-        .expect("must execute step request");
+    builder.advance_eras_by(
+        1,
+        [
+            RewardItem::new(NON_FOUNDER_VALIDATOR_1_PK.clone(), 1),
+            RewardItem::new(NON_FOUNDER_VALIDATOR_2_PK.clone(), 1),
+        ]
+        .into(),
+    );
```

Comment:
As @Fraser999 noted, it seems like it's only the case for when we advance by a single era that we want to add the rewards. Let's change the API of that method then (`advance_eras_by_default_auction_delay`) and please, use `vec![...]` rather than `[...].into()`.

Reply:
Looks like we have one counter example in bids.rs line 3890.

Made the change to vec! instead of into().

Reply:
This one:

https://github.com/HansWestman74/casper-node-1/blob/a1ad558f07f8384ec73a7ecaca56755d3f1e50b1/execution_engine_testing/tests/src/test/system_contracts/auction/bids.rs#L3900